src/plugins/console/public/lib/autocomplete_entities/autocomplete_entities.test.js

```diff
@@ -119,6 +120,88 @@ describe('Autocomplete entities', () => {
         expect(fields).toEqual([{ name: '@timestamp', type: 'date' }]);
       });
 
+      test('caches mappings for wildcard requests', async () => {
+        httpMock.get.mockReturnValue(
+          Promise.resolve({
+            mappings: {
+              'my-index-01': { mappings: { properties: { '@timestamp': { type: 'date' } } } },
+              'my-index-02': { mappings: { properties: { name: { type: 'keyword' } } } },
+            },
+          })
+        );
+
+        const autoCompleteContext = {};
+
+        mapping.getMappings('my-index*', [], autoCompleteContext);
+
+        const fields = await autoCompleteContext.asyncResultsState.results;
+        expect(fields).toEqual([
+          {
+            name: '@timestamp',
+            type: 'date',
+          },
+          {
+            name: 'name',
+            type: 'keyword',
+          },
+        ]);
+      });
+
+      test('returns mappings for data streams', () => {
+        dataStream.loadDataStreams({
+          data_streams: [
+            { name: 'test_index1', indices: [{ index_name: '.ds-index-1' }] },
+            {
+              name: 'test_index3',
+              indices: [{ index_name: '.ds-index-3' }, { index_name: '.ds-index-4' }],
+            },
+          ],
+        });
+        mapping.loadMappings({
+          '.ds-index-3': {
+            properties: {
+              first_name: {
+                type: 'string',
+                index: 'analyzed',
+                path: 'just_name',
+                fields: {
+                  any_name: { type: 'string', index: 'analyzed' },
+                },
+              },
+              last_name: {
+                type: 'string',
+                index: 'no',
+                fields: {
+                  raw: { type: 'string', index: 'analyzed' },
+                },
+              },
+            },
+          },
+        });
+
+        const result = mapping.getMappings('test_index3', []);
+        expect(result).toEqual([
+          {
+            name: 'first_name',
+            type: 'string',
+          },
+          {
+            name: 'any_name',
+            type: 'string',
+          },
+          {
+            name: 'last_name',
+            type: 'string',
+          },
+          {
+            name: 'last_name.raw',
+            type: 'string',
+          },
+        ]);
+      });
+
+      test('returns mappings for wildcards', () => {});
```

Comment:
nit: empty test

Reply:
updated the other test `caches mappings for wildcard requests` and removed this empty one in https://github.com/elastic/kibana/pull/150668/commits/7e80a331ec45146a0b46a3ba3bbda5ce00c14c17